src/run/runner/wall_time/helpers.rs

```diff
@@ -41,9 +46,12 @@ pub fn validate_walltime_results(profile_folder: &Path) -> Result<()> {
             .with_context(|| format!("Failed to parse walltime results from: {path:?}"))?;
 
         if results.benchmarks.is_empty() {
-            bail!(add_empty_result_error_explanation(&format!(
-                "No benchmarks found in walltime results file: {path:?}."
-            )));
+            if !allow_empty {
+                bail!(add_empty_result_error_explanation(&format!(
+                    "No benchmarks found in walltime results file: {path:?}."
+                )));
+            }
+            debug!("No benchmarks found in {path:?} (allowed)");
         }
 
         found_valid_results = true;
```

Comment:
Since the code will bail if there is a parsing error, let's rename this variable to `found_results` so that it is not misleading